install.dd

```diff
@@ -0,0 +1,163 @@
+Ddoc
+
+
+$(D_S $(TITLE),
+
+$(UL
+    $(LI $(RELATIVE_LINK2 get, Getting the script))
+    $(LI $(RELATIVE_LINK2 usage, Usage))
+    $(UL
+        $(LI $(RELATIVE_LINK2 install, Installation))
+        $(LI $(RELATIVE_LINK2 uninstall, Uninstallation))
+        $(LI $(RELATIVE_LINK2 list, Listing available compilers))
+        $(LI $(RELATIVE_LINK2 update, Update))
+    )
+)
+
+$(H2 $(LNAME2 get, Getting the script))
+
+$(CONSOLE
+wget https://dlang.org/install.sh
+)
+
+If you want, you can directly invoke it:
+
+$(CONSOLE
+curl https://dlang.org/install.sh | bash -s
+)
+
+$(P If no argument are provided, the latest DMD compiler will be installed.)
+
+$(P The installer will also install a copy of itself to `~/dlang/install.sh`.)
+
+$(H2 $(LNAME2 usage, Usage))
+
+$(CONSOLE
+./install.sh [&lt;command&gt;] [&lt;args&gt;]
```

Comment:
Always use `~/dlang/install.sh`. Avoid using relative paths, especially in documentation.